### PR TITLE
Override etcd timeouts with cli args

### PR DESCRIPTION
**Pull request by mpeters**
Also adding more debug statements

### CODE REVIEW

File: provider/etcd/etcd_client.go [hunk 1]
```diff
@@ -362,3 +368,17 @@ func UnmarshalAlert(alertStr string) (*types.Alert, error) {
 	}
 	return &alert, nil
 }
+
+func getDurationFromEnv(envVar string, defaultVal int) time.Duration {
+	envVal, exists := os.LookupEnv(envVar)
+	if exists {
+		intVal, err := strconv.Atoi(envVal)
+		if err != nil {
+			return time.Duration(defaultVal) * time.Millisecond
```
Comment (hangxie) [inline provider/etcd/etcd_client.go]: MAY write some warning messages, it helps people correct their mistakes.

**Review by hangxie — APPROVED** [review]
☕️

File: provider/etcd/etcd_client.go (hunk 1 above)
Comment (mpeters) [reply provider/etcd/etcd_client.go]: I was thinking about that too, but this happens before we have access to the logger object attached to the alerts. I'm sure there's a way, but I'm new to golang, so it wasn't obvious to me

**Comment by dcwangmit01** [comment]
Looks mergeable.  Note that this kind of thing in alertmanager is usually command line args, if you wanted to be consistent.  Not a big deal.

**Comment by dcwangmit01** [comment]
\+ 🔵

**Pull request merged**